src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const CAR_RENT = 40;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
```

Comment:
Move CONSTANTS outside the function.
CONSTANT - This is a variable that is created only 1 time and never changes.
In ur case, it is created every time the function is called